scripts/bug_classifier.py

```diff
@@ -22,9 +23,12 @@ def classify_bugs(model_name, classifier):
     else:
         model_file_name = f"{model_name}model"
 
-    assert os.path.exists(
-        model_file_name
-    ), f"{model_file_name} does not exist. Train the model with trainer.py first."
+    if not os.path.exists(model_file_name):
+        print(f"{model_file_name} does not exist. Downloading the model....")
+        download_url = f"https://index.taskcluster.net/v1/task/project.relman.bugbug.train_{model_name}.latest/artifacts/public/{model_file_name}.zst"
```

Comment:
Nit: I'd remove this variable and replace its usage with its value
`download_check_etag(f"https://index.tas...", f"{model_file...")`